src/main/java/ithillel/lesson5/ArrayHillelList.java

```diff
@@ -0,0 +1,55 @@
+package ithillel.lesson5;
+
+public class ArrayHillelList implements HillelList{
+
+    private String[] ArrayList = {};
```

Comment:
```suggestion
    private String[] elements = {};
```
elements, data, items - будуть кращіми назвами тут 

ArrayList - не дуже підходить по двом причинам: такий клас в Java вже є і не змінні мають називатися з маленької букви (arrayList)